apple/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -25,10 +25,11 @@ typedef NS_ENUM(NSInteger, DeviceType) {
     DeviceTypeHandset,
     DeviceTypeTablet,
     DeviceTypeTv,
+    DeviceTypeMac,
     DeviceTypeUnknown
 };
 
-#define DeviceTypeValues [NSArray arrayWithObjects: @"Handset", @"Tablet", @"Tv", @"unknown", nil]
+#define DeviceTypeValues [NSArray arrayWithObjects: @"Handset", @"Tablet", @"Tv", @"Mac", @"unknown", nil]
```

Comment:
I'm going to handle this one differently - #1137 added ability to return Desktop as a device type, and that is vendor-neutral - I want to return that for macos platform builds

The combo of Platform === iOS and DeviceType === Desktop === Catalyst builds
The combo of Platform === macOS and DeviceType === Desktop will === Desktop as well

Reply:
Sounds good!